share/github-backup-utils/ghe-backup-config

```diff
@@ -304,3 +304,28 @@ ghe_debug() {
 version() {
   echo "${@#v}" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }';
 }
+
+# The list of gists returned by the source changed in 2.16.23, 2.17.14,
+# 2.18.8, and 2.19.3.  So we need to account for this difference here.
+fix_paths_for_ghe_version() {
+  ruby -e '
```

Comment:
This'll introduce a dependency on Ruby for the backup-host, so will require updates to https://github.com/github/backup-utils/blob/master/docs/requirements.md#backup-host-requirements.